spec.html

```diff
@@ -32093,6 +32093,12 @@ <h1>Array.prototype.flat ( [ _depth_ ] )</h1>
         <emu-clause id="sec-flattenintoarray" aoid="FlattenIntoArray">
           <h1>FlattenIntoArray ( _target_, _source_, _sourceLen_, _start_, _depth_ [ , _mapperFunction_, _thisArg_ ] )</h1>
           <emu-alg>
+            1. Assert: Type(_target_) is Object.
+            1. Assert: Type(_source_) is Object.
+            1. Assert: _sourceLen_ is an integer Number &ge; 0.
```

Comment:
```suggestion
            1. Assert: _sourceLen_ is an integral Number &ge; 0.
```

Reply:
hm, i used the same terminology used elsewhere in the spec (plus imo “integral” isn’t unambiguously “integer”)

Reply:
The current spec has 3 occurrences of "integer Number" and none of "integral Number".

We could change "integer Number" to just "integer", now that it's [defined](https://tc39.es/ecma262/#integer).

As for `&ge; 0` vs `is nonnegative`: the spec uses both, but in algorithm steps, it uses `&ge; 0` more. (I'd say that, in a context where either would fit, you should pick the symbolic, because it's shorter and [it seems to me] easier to understand.)

(Side note: the spec has only one occurrence of "non-negative" with the hyphen. Looks like there was a purge in ES6. Not sure why, because the spec certainly has no problem with the hyphen in terms like non-configurable, non-strict, and non-object.)

Reply:
I'm going to resolve this one for this PR, but please file an issue/PR to make this kind of thing consistent throughout.